src/braket/jobs/quantum_job_creation.py

```diff
@@ -178,8 +179,8 @@ def prepare_quantum_job(
             "compressionType": "GZIP",
         }
     }
-    if image_uri:
-        algorithm_specification["containerImage"] = {"uri": image_uri}
+    image_uri = image_uri or image_uris.retrieve_image(Framework.BASE, aws_session.region)
```

Comment:
It's a lot easier to update the service then updating all the clients. So, for example, if we do this, people with older clients won't be updated.

Reply:
Good catch, updating the uris returned by this function to get the latest version in https://github.com/aws/amazon-braket-sdk-python/pull/644